### PR TITLE
👷 Refactor ➾ (ci) move tests to self-hosted Mac Minis

### PR DESCRIPTION
**Pull request by GImbrailo**
# 🌮 Taqueria PR

## 🪁 Description

We want to run our E2E tests on our self-hosted runners on Mac Minis. The tests run significantly faster on the self-hosted runners.

## 🪂 Pre-Merge Checklist (Definition of Done)

🚦 Required to merge:

- [x] ⛱️ I have completed this PR template in full and updated the title appropriately
- [x] ⛵ My code builds cleanly, and I have manually tested the changes

## 🛩️ Summary of Changes

- [x] Change the runner tag from `macOS-latest` to `ecad-macmini` for the `run-tests` job
- [x] Remove caching as this often has a negative impact on the workflows and can make them take longer due to network restraints
- [x] Remove the `--maxWorkers=1` option for e2e tests to run tests in parallel. 

## 🛸 Type of Change

- [x] 👷 Refactor ➾

### CODE REVIEW

File: .github/workflows/vscode-extension.yml [hunk 1]
```diff
@@ -57,11 +57,13 @@ jobs:
...
-        uses: actions/cache@v3
-        with:
-          path: ./node_modules/**
-          key: cache-node-modules-${{ github.run_id }}
+      - name: Add working dir to PATH
+        run: echo "$(pwd)" >> $GITHUB_PATH
+
+      - name: Build all
+        run: |
+          npx lerna bootstrap
```
Comment (danielelisi) [inline .github/workflows/vscode-extension.yml:65]: I'd suggest adding `--force-local` to ensure local packages are linked

**Review by danielelisi — APPROVED** [review]
Just small comment for `lerna bootstrap`. Other than than LGTM.

**Comment by danielelisi** [comment]
I see one of the build `main / OS ubuntu-latest test chunk 0 (pull_request) ` failing. Is that expected?

**Review by hu3man — APPROVED** [review]
Super excited to see this go live, great work @GImbrailo !

**Pull request merged**